DXRando/DeusEx/Classes/InformationDevices.uc

```diff
@@ -318,6 +318,24 @@ function bool Facelift(bool bOn)
 }
 #endif
 
+function Frob(actor Frobber, Inventory frobWith)
+{
+    Super.Frob(Frobber, frobWith);
+    GlowOff();
+}
+
+final function bool GlowOff()
+{
+    local DynamicLight lt;
+
+    if (#var(prefix)DataCube(self) != None) {
+        foreach RadiusActors(class'DynamicLight', lt, 10.0) {
```

Comment:
I think the DynamicLight will be one of the `BasedActors` for the datacube? that might be a little cleaner, especially when multiple datacubes are close together